tools/openms/MapAlignerPoseClustering.xml

```diff
@@ -22,12 +22,12 @@
 
 -out
 #for $counter,$infile in enumerate($param_in):
-    ./results_out/${infile.element_identifier}.${infile.extension}
+    './results_out/${infile.element_identifier}.${infile.extension}'
```

Comment:
@jvolkening you might want to do this in addition: 
`#set escaped_element_identifier = re.sub('[^\w\-\s]', '_', str($bam.element_identifier))`